### ISSUE TITLE
hyperparameter selection

### ISSUE THREAD

**Issue by hzhz2020** [guolz-ml/DS3L#6]
Dear Authors
Thank you for your fundamental work in the robust SSL.
Regarding the experiment setting, how did you select the hyperparameters? Did you use a validation set or just adopt the hyperparameters from some prior literature? 

Looking forward to hearing from you!
